### PR TITLE
Minor code cleanup

### PR DESCRIPTION
**Pull request by ajs6f**
https://jira.duraspace.org/browse/FCREPO-1603

### CODE REVIEW

File: fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/domain/SinglePrefer.java [hunk 1]
```diff
@@ -84,15 +84,7 @@ public PreferTag getHandling() {
...
-    private static final HttpHeaderReader.ListElementCreator<PreferTag> PREFER_CREATOR =
-            new HttpHeaderReader.ListElementCreator<PreferTag>() {
-
-                @Override
-                public PreferTag create(final HttpHeaderReader reader) throws ParseException {
-                    return new PreferTag(reader);
-                }
-            };
-
+    private static final HttpHeaderReader.ListElementCreator<PreferTag> PREFER_CREATOR = r -> new PreferTag(r);
```
Comment (awoods) [inline fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/domain/SinglePrefer.java]: This would also work, no?
`PREFER_CREATOR = PreferTag::new;`

Comment (ajs6f) [inline fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/domain/SinglePrefer.java]: Nice, yes. Maybe just in-line it?

Comment (awoods) [inline fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/domain/SinglePrefer.java]: sure

File: fcrepo-kernel/src/main/java/org/fcrepo/kernel/utils/UncheckedBiConsumer.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    }
+
+    /**
+     * The same semantic as {@link #accept(Object, Object)}, but allowed to throw exceptions.
+     *
+     * @param first
+     * @param second
+     * @throws Exception
+     */
+    void acceptThrows(final S first, final T second) throws Exception;
```
Comment (awoods) [inline fcrepo-kernel/src/main/java/org/fcrepo/kernel/utils/UncheckedBiConsumer.java]: This can safely be `private`, no?

Comment (ajs6f) [inline fcrepo-kernel/src/main/java/org/fcrepo/kernel/utils/UncheckedBiConsumer.java]: This is an interface. There are no `private` methods.

**Comment by awoods** [comment]
Resolved with: https://github.com/fcrepo4/fcrepo4/commit/e8b3f63b3dae8e25c73e6bec4a24ebd6564e130f

**Pull request closed without merging**